concepts/observability-strategy/opentelemetry-operator/README.md

```diff
@@ -0,0 +1,89 @@
+# OpenTelemetry Operator PoC
+
+## About
+A crucial aspect which was neglected to much in the previous concept design is the feature of activating instrumentation and exporter configuration of a workload in a seamless fashion. That feature is already available in the [opentelemetry-operator](https://github.com/open-telemetry/opentelemetry-operator#opentelemetry-auto-instrumentation-injection). Via central configuration you can inject relevant configuration by putting a simple annotation. Relevant programming languages are supported by auto-instrumentation already.
+
+Re-inventing the wheel should be avoided, so the PoC discusses the impact on introducing the operator in irder to provide the mentioned features to the users.
+
+## Scope
+Introducing the operator to the stack either as underlying technology only or actual feature, the effect will be that the CRDs will be visible to runtime users and users can independently use them. Trying to isolate them will not be of success as previous experience with the prometheus-operator shows that this will only cause conflicts with potential operator installations provided by the user. Meaning of: restricting the operator usage in the runtime AND blocking the installation of a custom operator installation is not an option as users cannot leverage the power of it at all. Running a restricted operator for internal usage and allowing to install a custom operator is resulting in many potential conflicts like race-reconcilations on the same resources or webhook conflicts (as custom operator is not excluding system namespaces).
+Conclusion: either the operator gets used as underlying technology for the kyma abstractions and instrumentation support AND is accessible for custom scenarios for users. Or the operator gets not used. No other option.
+
+## Proposal
+
+- The existing `telemetry-operator` manages an opentelemetry-operator additionally.
+- The current abstraction `TracePipeline` does not reveal into an actual deployment but an `OpenTelemetryCollector` resource configuration. The configuration gets reconciled and with that cannot be influenced by the user. The `opentelemetry-collector` will resolve the configuration into an actual deployment. That part is still optional and the indirection via the opentelemetry-operator is not mandatory. It will be mainly beneficial for the user to copy-over sich configuration in order to realize a custom trace-collector
+- The telemetry-operator reconciles `Instrumentation` resources which can be used by the user instantly for auto-instrumentation and injection of the `trace-collector` reference
```

Comment:
```suggestion
- The Telemetry operator reconciles `Instrumentation` resources, which can be used by the user instantly for auto-instrumentation and injection of the `trace-collector` reference
```

Again, I believe we use natural language when there are no backticks, and then Telemetry (our component) should be capitalized.